clients/cloud/rust/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "confluent_cloud"
```

Comment:
confluent_cloud_example to avoid confusion with future confluent_cloud packages (whatever that would be)?